flax/linen/fp8_ops.py

```diff
@@ -259,6 +269,7 @@ def dot_general_with_precision_jvp(
 
 
 class Fp8DotGeneralOp(module.Module):
+  fp8_genre: str = 'OCP'
```

Comment:
instead of indirecting through a string lookup in `get_fp8_dtypes`, can we please just make the fwd/bwd fp8 dtypes class attributes and default to existing "ocp" types, and place the new fields _after_ the already existing one
```
amax_history_length: int = 1024
e4m3_dtype: Dtype = jnp.float8_e4m3fn
e5m2_dtype: Dtype = jnp.float8_e5m2
```

(feel free to define a typing alias dtype up top:
```
from typing import Any
Dtype = Any 
```
)

Reply:
Done.